xcp_d/cli/run.py

```diff
@@ -153,14 +153,6 @@ def main():
                 f"boilerplate text found in {boiler_file}.",
             )
 
-        if config.workflow.run_reconall:
```

Comment:
This was the problem- `config.workflow.run_reconall` is an attribute that doesn't exist in the XCP-D Config, but its location within this try/except/else/finally statement meant that it wasn't doing anything except blocking the line that sets `errno` to 0.